### PR TITLE
fix(babel-preset-mc-app): remove uncalled-for plugins

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

With the latest release of babel some of the plugins we're still using are removable.

CleanShot 2020-01-20 at 09 03 16@2x

### CODE REVIEW

File: packages/babel-preset-mc-app/index.js [hunk 1]
```diff
@@ -127,10 +127,7 @@ module.exports = function getBabePresetConfigForMcApp() {
         },
       ],
       require('@babel/plugin-proposal-do-expressions').default,
-      require('@babel/plugin-proposal-optional-chaining').default,
-      require('@babel/plugin-proposal-nullish-coalescing-operator').default,
       // Adds syntax support for import()
-      require('@babel/plugin-syntax-dynamic-import').default,
```
Comment (emmenko) [inline packages/babel-preset-mc-app/index.js]: Ah you removed this but remove the dependency of `babel-plugin-transform-dynamic-import`.

Which one should we remove?

Comment (tdeekens) [reply packages/babel-preset-mc-app/index.js]: Good point. The changelog is not the most expressive on this. 

CleanShot 2020-01-20 at 09 38 58@2x

Somehow it sounds to me as the support it natively now. I removed both. Will build and give it a spin locally and see what happens.

Comment (tdeekens) [reply packages/babel-preset-mc-app/index.js]: So the tests locally with this version of the preset pass. Also a production build can be made. Anything else besides the later canary build we should test before moving on?

Comment (emmenko) [reply packages/babel-preset-mc-app/index.js]: If everything passes, it should be fine

File: .vscode/settings.json [hunk 2]
```diff
@@ -18,5 +18,8 @@
     }
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale"
+  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale",
+  "workbench.colorCustomizations": {
+    "tab.unfocusedActiveBorder": "#fff0"
```
Comment (emmenko) [inline .vscode/settings.json]: 😔

I guess we need to git ignore `settings.json` file again

Comment (tdeekens) [reply .vscode/settings.json]: FF*. Every lovely morning day. I commit the bloody settings.json here.

**Pull request merged**